PyBambooHR/PyBambooHR.py

```diff
@@ -628,24 +656,105 @@ def get_whos_out(self, start_date=None, end_date=None):
         # return utils.transform_whos_out(r.content)
 
     def get_time_off_requests(self, start_date=None, end_date=None, status=None, type=None, employee_id=None):
+        """
+        API method for returning a list of time off requests
+        https://www.bamboohr.com/api/documentation/time_off.php#getRequests
+        Success response: 200
+        @return: List containing time off request dictionaries
+        """
         start_date = utils.resolve_date_argument(start_date)
         end_date = utils.resolve_date_argument(end_date)
 
         params = {}
-        if start_date:
-            params['start'] = start_date
-        if end_date:
-            params['end'] = end_date
-        if status:
-            params['status'] = status
+        # Must set dates or request errors 400 Bad Request
```

Comment:
BambooHR's API says it is optional. :thinking: 
https://www.bamboohr.com/api/documentation/time_off.php#getRequest
> start (optional) - Only show time off that occurs on/after the specified start date.
> end (optional) - Only show time off that occurs on/before the specified end date.